src/plugins/postal-savings-rs/preferences.xml

```diff
@@ -19,6 +19,16 @@
         negativeButtonText="Cancel"
         summary="||***********"
     />
+    <EditTextPreference
+        key="cardNumber"
```

Comment:
I haven't found a way to get cards numbers from bank, so we have to specify it manually to be able to fetch card transactions.
I bet, one field for card number covers 99% of users, but it would be better to somehow add more than one card and bind cards to corresponding accounts.

Reply:
Need to think about backwards compatibility: users don't have this new field in settings.
Also need to check if some users may not have a card at all.

Reply:
Replaced this field with user input (see the first commit)